index.html

```diff
@@ -30,12 +22,19 @@
           </li>
         </ul>
         <a href="#" class="mobile-nav">
-          <img
-            class="navImage"
-            src="./Images/ic_menu.svg"
-            alt="Navigation menu"
-          />
+          <img class="navImage" src="./Images/ic_menu.svg" alt="Navigation menu" />
         </a>
+        <div class="header__menu">
+          <button class="header__menu-btn">
+            <img src="images/Icon-Cancel.png" alt="Menu cancel button" />
+          </button>
+
+          <ul class="header__menu-content">
+            <li><a class="header__menu-link" href="#portfolio">Portfolio</a></li>
+            <li><a class="header__menu-link" href="#about">About</a></li>
+            <li><a class="header__menu-link" href="#contact">Contact</a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Please make sure that the mobile menu takes 100% of the screen's height to match the given design.

![image](https://user-images.githubusercontent.com/62290545/208647873-005550e2-2450-4bd1-9423-3ac32ceda8db.png)